### PR TITLE
feat: support swift package manager

### PR DESCRIPTION
**Pull request by s2mr**
https://github.com/bucketeer-io/ios-client-sdk/issues/2

### CODE REVIEW

File: Bucketeer/Sources/Internal/Utils/Version.swift [hunk 1]
```diff
@@ -0,0 +1,3 @@
+enum Version {
+    static let current = "2.0.0"
```
Comment (cre8ivejp) [inline Bucketeer/Sources/Internal/Utils/Version.swift]: I think we will need to add the comment to this line so the CI can update it automatically before releasing it.

E.g.
```swift
static let current = "2.0.0" // x-release-please-version
```

https://github.com/bucketeer-io/ios-client-sdk/blob/main/Bucketeer.podspec#L3

And then add the file path here.

https://github.com/bucketeer-io/ios-client-sdk/blob/main/.github/workflows/release.yml#L28

Comment (s2mr) [reply Bucketeer/Sources/Internal/Utils/Version.swift]: Sorry, I didn't know the release flow.
Thank you, I fixed it.

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import PackageDescription
+
+let package = Package(
+    name: "Bucketeer",
+    platforms: [.iOS(.v11)],
+    products: [
+        .library(
+            name: "Bucketeer",
+            targets: ["Bucketeer"]
+        ),
```
Comment (cre8ivejp) [inline Package.swift]: Can you fix the lint warning below?

```
Trailing Comma Violation: Collection literals should not have trailing commas (trailing_comma)
```

**Review by cre8ivejp — APPROVED** [review]
Thank you for your contribution!!! 🎉

**Pull request merged**